dvc/remote/azure.py

```diff
@@ -26,6 +26,7 @@ class RemoteAZURE(RemoteBASE):
         r"(?P<connection_string>.+)?)?)$"
     )
     REQUIRES = {"azure-storage-blob": "azure.storage.blob"}
+    DEFAULT_NO_TRAVERSE = False
```

Comment:
Do we still need this flag per-remote? Looks like we have a default automatic detection now. Should we also keep an ability to explicitly set no_traverse in config?

Reply:
I was thinking that if we leave the `no_traverse` config option, users who are sure that `no_traverse = true` will always be faster for their particular use case can set it to avoid the auto-detection step entirely.